libs/RobotKit/include/RobotController.h

```diff
@@ -203,6 +203,12 @@ class RobotController : public interface::RobotController
 		system_reset();
 	}
 
+	void turnOffActuators() final
+	{
+		_lcd.turnOff();
+		stopActuators();
+	}
+
```

Comment:
est-ce qu'on a besoin d'une nouvelle fonction ici? on ne peut pas mettre le lcd dans `stopActuators`?

si on peut pas, il faut un nom qui montre la différence et les mettre l'une à côté de l'autre dans le code (celle-ci en dessous)

```suggestion
	void stopActuatorsAndLcd() final
	{
		_lcd.turnOff();
		stopActuators();
	}

```